tests/integration_test/src/tests/seccomp/mod.rs

```diff
@@ -0,0 +1,49 @@
+use oci_spec::runtime::{
+    Arch, LinuxBuilder, LinuxSeccomp, LinuxSeccompAction, LinuxSeccompBuilder, LinuxSyscallBuilder,
+    ProcessBuilder, Spec, SpecBuilder,
+};
+use test_framework::{Test, TestGroup, TestResult};
+
+use crate::utils::test_inside_container;
+
+fn create_spec(seccomp: LinuxSeccomp) -> Spec {
+    SpecBuilder::default()
+        .linux(
+            LinuxBuilder::default()
+                .seccomp(seccomp)
+                .build()
+                .expect("error in building linux config"),
+        )
+        .process(
+            ProcessBuilder::default()
+                .args(vec!["runtimetest".to_string(), "seccomp".to_string()])
+                .build()
+                .expect("error in creating process config"),
+        )
+        .build()
+        .unwrap()
+}
+
+fn seccomp_test() -> TestResult {
+    let spec = create_spec(
+        LinuxSeccompBuilder::default()
+            .default_action(LinuxSeccompAction::ScmpActAllow)
+            .architectures(vec![Arch::ScmpArchX86_64])
```

Comment:
Do we need to setup the arch specifically, or because we are using the build pattern, we can not specify it and let the builder take default one?

Reply:
It seems that we don't need to explicitly set this option in the builder.